2018 Robot/src/org/usfirst/frc/team4999/commands/autonomous/TurnDegrees.java

```diff
@@ -0,0 +1,87 @@
+package org.usfirst.frc.team4999.commands.autonomous;
+
+import org.usfirst.frc.team4999.robot.MoPrefs;
+import org.usfirst.frc.team4999.robot.Robot;
+import org.usfirst.frc.team4999.robot.RobotMap;
+import org.usfirst.frc.team4999.robot.sensors.GyroFusion;
+import org.usfirst.frc.team4999.robot.subsystems.DriveSystem;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.PIDOutput;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.PIDSourceType;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class TurnDegrees extends Command {
+
+	private Encoder left = RobotMap.leftDriveEncoder;
+	private Encoder right = RobotMap.rightDriveEncoder;
+	
+	private double ticksPerMeter = 1;
```

Comment:
I've now seen this same constant in 2 files.  Absolutely not ok.  Must be centralized somewhere.